app/helpers/checkouts_helper.rb

```diff
@@ -2,30 +2,7 @@
 
 # Helper for checkouts views
 module CheckoutsHelper
-  def render_checkout_status(checkout)
-    return unless checkout.overdue?
-
-    checkout.claims_returned? ? 'Claims Returned' : 'Overdue'
-  end
-
-  def render_checkout_due_date(checkout)
-    contents = []
-    if checkout.recalled?
-      contents << 'Recalled'
-      contents << format_due_date(checkout.recall_due_date)
-    end
-    contents << format_due_date(checkout.due_date)
-
-    content_tag 'span', contents.join('<br>'), nil, false
-  end
-
-  def format_due_date(date)
-    return l(date, format: :long) unless l(date, format: :time_only) == '11:59pm'
-
-    l(date, format: :short)
-  end
-
   def render_renew_button
-    submit_tag 'Renew', class: 'btn btn-primary btn-renewable-submit'
+    submit_tag 'Renew', class: 'btn btn-primary', data: { disable_with: 'Please wait...' }
   end
```

Comment:
What do you think of removing this helper method too to finish off the complete removal? Looks like it's only used once and doesn't do much special process, just a Rails tag method call.... 🧨

Reply:
I thought about that, too and could not decide coz the same renew button is used both top and bottom of the table so twice 🤔

Reply:
Yeah I am. The reason is that I think we should avoid using helpers at all if it can be avoided, and the use case for when it is used should include some more complicated logic or processing. I think I'd rather skip the helper and just have two identical submit tags. What do you think?

Reply:
got the helper method removed 👍